mappings/net/minecraft/entity/passive/FoxEntity.mapping

```diff
@@ -122,7 +122,7 @@ CLASS net/minecraft/class_4019 net/minecraft/entity/passive/FoxEntity
 	CLASS class_4024 MateGoal
 		METHOD <init> (Lnet/minecraft/class_4019;D)V
 			ARG 2 chance
-	CLASS class_4025 EatSweetBerriesGoal
+	CLASS class_4025 EatBerriesGoal
```

Comment:
The `eatSweetBerry` method within this class should also be changed, as it applied to glow berries too.